src/main/scala/io/altoo/akka/serialization/kryo/KryoSerializer.scala

```diff
@@ -207,6 +207,8 @@ class KryoSerializer(val system: ExtendedActorSystem) extends Serializer {
     // if explicit we require all classes to be registered explicitely
     if (strategy == "explicit")
       kryo.setRegistrationRequired(true)
```

Comment:
one setRegRequired is enough

Reply:
@danischroeter default changed in Kryo 5, since it depends on our strategy we have to set it either way to be sure. Could be done in one line though.